EpiAware/src/EpiObsModels/EpiObsModels.jl

```diff
@@ -11,6 +11,10 @@ using ..EpiLatentModels: HierarchicalNormal, broadcast_dayofweek
 
 using Turing, Distributions, DocStringExtensions, SparseArrays
 
+# Abstract observation model
+export AbstractTuringObservationErrorModel, generate_observation_error_priors,
```

Comment:
I thought we were keeping a code pattern of types and methods being exported in separate lines for readability?

Reply:
yes though this was just to group this discrete part. The main question I have here is should we move the abstract type to base as well as the non-Turing method?